orderbook/src/database/orders.rs

```diff
@@ -120,6 +129,11 @@ impl OrderStoring for Postgres {
             .bind(order.order_creation.partially_fillable)
             .bind(order.order_creation.signature.to_bytes().as_ref())
             .bind(DbSigningScheme::from(order.order_creation.signing_scheme))
+            // TODO - find a non-blocking way of accessing this.
+            .bind("0x3328f5f2cEcAF00a2443082B657CedEAf70bfAEf")
```

Comment:
Only thing that remains here is to find a non-blocking way to access this. Does anyone have any clever suggestions that doesn't involve putting the settlement_address as a new field on the Postgres instance and adapting the constructor in the 20 places it is used throughout the code?

Reply:
I have offloaded this adjustment to a separate PR where we can discuss the best approach.

https://github.com/gnosis/gp-v2-services/pull/907